Tools-Override/tests.targets

```diff
@@ -28,13 +28,15 @@
 
   <PropertyGroup>
     <XunitRuntimeConfig>$(ToolsDir)\xunit.console.netcore.runtimeconfig.json</XunitRuntimeConfig>
-    <TestRuntimeEnvVar Condition="'$(OS)' == 'Windows_NT'">%RUNTIME_PATH%\</TestRuntimeEnvVar>
+    <TestRuntimeEnvVar Condition="'$(OS)' == 'Windows_NT' and ('$(BuildingInsideVisualStudio)' != 'true' or '$(RUNTIME_PATH)' != '')">%RUNTIME_PATH%\</TestRuntimeEnvVar>
     <TestRuntimeEnvVar Condition="'$(OS)' != 'Windows_NT'">$RUNTIME_PATH/</TestRuntimeEnvVar>
+    <TestRuntimeEnvVar Condition="'$(TestRuntimeEnvVar)' == ''">$(TestSharedFxDir)\</TestRuntimeEnvVar>
```

Comment:
I understand the desire to keep these in sync. This is tough because we're trying to support 3 distinct ways to run tests, with slightly different calling styles. I don't have a big objection to this approach. Another thing to note, though, is that the RunTests.cmd script is going to inherit the absolute paths here, when you build in VS. I doubt that will ever cause a problem for anyone, but it is a slight quirk.